tasks/WebRTCReceiveTask.cpp

```diff
@@ -218,6 +222,7 @@ void WebRTCReceiveTask::onIncomingStream(GstElement* webrtcbin, GstPad* pad)
     gst_bin_add(GST_BIN(m_pipeline), bin);
 
     GstElement* decodebin = gst_element_factory_make("decodebin", NULL);
+    g_object_set(decodebin, "max-size-buffers", 1, NULL);
```

Comment:
As far as I remember, this did not change much. I would remove it, I think it can cause problems.